TIL/daily/2025-03-21.md

```
@@ -0,0 +1,245 @@
+bool operator ==(Object other) =>
+	identical(this, other) ||
+	other is Hero && runtimeType == other.runtimeType;
+```
+
+### hashCode 재정의
+
+- hashCode를 재정의하면 Set, Map 내부에서의 동등성 비교가 가능하다
+
+```dart
+class Person {
+	String name;
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)